src/components/button/Primary/container.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode } from "react";
+import { constantsButton } from "..";
+import { constants } from "../../../typography";
+import { colors } from "../../../typography/colors";
+
+const stylesButton = {
+  width: constantsButton.primary.width,
+  fontFamily: constants.font.family.primary,
+  fontSize: constantsButton.primary.size,
+  color: colors.Neutral,
+  fontWeight: constantsButton.primary.weight,
+  borderRadius: constantsButton.primary.radius,
+  border: constantsButton.primary.border,
+};
+interface buttonProps {
+  type?:
+    | "default"
+    | "defaultCompact"
+    | "disableCompact"
+    | "disableDefault"
+    | "hover";
+

```

Comment:
Acho que vale vc refletir um pouco mais em como traduzir essas propriedades que vc viu no Figma em código. Os tipos dos botões são os que estão nas colunas: primário, secundário e CTA (call-to-action). Esse tipo define um conjunto de estilos que faz um botão ser daquele tipo. Por exemplo, o primário tem aquele fundo azul escuro e a label branca.

As propriedades `disable` e `compact` são características que qualquer um dos 3 tipos de botões podem ter, dependendo da sua utilização.

E a variação hover também é algo que todos os botões tem. Mas diferente das duas anteriores, que quem decide é quem está usando o botão, o hover é uma característica inata/interna, toods os botões tem um estado de hover, você não precisa informar via props que ele tá em hover, certo?